src/styles/style.css

```diff
@@ -0,0 +1,30 @@
+/* add styles here */
+
+body {
+  margin: 0;
+}
+
+.search_bars {
+  padding: 0 41px;
+  display: inline-block;
+  box-sizing: border-box;
+}
+
+.first_search--style{
+  width: 900px;
+  height: 70px;
+  margin-top: 71px;
+  margin-bottom: 50px;
+  border:solid 1px #E1E7ED;
+  border-radius: 4px;
+
+}
+
+.second_search--style{
+  padding-bottom: 20px;
+  background-color:#2F4F4F;
+  border:solid 1px;
+  border-radius:15px;
+  width:300px;
```

Comment:
the search input should have full width, to accomplish that we need to set width to 100%